dev/Breadcrumb/BreadcrumbBarItem.cpp

```diff
@@ -528,32 +532,28 @@ void BreadcrumbBarItem::InstantiateFlyout()
     // Only if the element has been created visually, instantiate the flyout
     if (const auto& button = m_button.get())
     {
-        // Create ItemsRepeater and set the DataTemplate 
-        const auto& ellipsisItemsRepeater = winrt::ItemsRepeater();
-        ellipsisItemsRepeater.Name(s_ellipsisItemsRepeaterPartName);
-        winrt::AutomationProperties::SetName(ellipsisItemsRepeater, s_ellipsisItemsRepeaterAutomationName);
-        ellipsisItemsRepeater.HorizontalAlignment(winrt::HorizontalAlignment::Stretch);
+        if (const auto& ellipsisFlyout = m_ellipsisFlyout.get())
+        {
+            // Create ItemsRepeater and set the DataTemplate 
+            const auto& ellipsisItemsRepeater = winrt::ItemsRepeater();
+            ellipsisItemsRepeater.Name(s_ellipsisItemsRepeaterPartName);
+            winrt::AutomationProperties::SetName(ellipsisItemsRepeater, s_ellipsisItemsRepeaterAutomationName);
+            ellipsisItemsRepeater.HorizontalAlignment(winrt::HorizontalAlignment::Stretch);
 
-        m_ellipsisElementFactory = winrt::make_self<BreadcrumbElementFactory>();
-        ellipsisItemsRepeater.ItemTemplate(*m_ellipsisElementFactory);
+            m_ellipsisElementFactory = winrt::make_self<BreadcrumbElementFactory>();
+            ellipsisItemsRepeater.ItemTemplate(*m_ellipsisElementFactory);
 
-        const auto& stackLayout = winrt::StackLayout();
-        stackLayout.Orientation(winrt::Controls::Orientation::Vertical);
-        ellipsisItemsRepeater.Layout(stackLayout);
+            if (const auto& dataTemplate = m_ellipsisDropDownItemDataTemplate.get())
+            {
+                m_ellipsisElementFactory->UserElementFactory(dataTemplate);
+            }
 
-        if (const auto& dataTemplate = m_ellipsisDropDownItemDataTemplate.get())
-        {
-            m_ellipsisElementFactory->UserElementFactory(dataTemplate);
-        }
+            m_ellipsisRepeaterElementPreparedRevoker = ellipsisItemsRepeater.ElementPrepared(winrt::auto_revoke, { this, &BreadcrumbBarItem::OnFlyoutElementPreparedEvent });
+            m_ellipsisRepeaterElementIndexChangedRevoker = ellipsisItemsRepeater.ElementIndexChanged(winrt::auto_revoke, { this, &BreadcrumbBarItem::OnFlyoutElementIndexChangedEvent });
 
-        m_ellipsisRepeaterElementPreparedRevoker = ellipsisItemsRepeater.ElementPrepared(winrt::auto_revoke, { this, &BreadcrumbBarItem::OnFlyoutElementPreparedEvent });
-        m_ellipsisRepeaterElementIndexChangedRevoker = ellipsisItemsRepeater.ElementIndexChanged(winrt::auto_revoke, { this, &BreadcrumbBarItem::OnFlyoutElementIndexChangedEvent });
-        
-        m_ellipsisItemsRepeater.set(ellipsisItemsRepeater);
+            m_ellipsisItemsRepeater.set(ellipsisItemsRepeater);
```

Comment:
is the style being applied? I remember having to do some style lookup so the style would be applied

Reply:
Do you mean for the flyout ? Yes, it is applied because the style is set on the template part and we are using that exact instance. 

---
In reply to: [588446070](https://github.com/microsoft/microsoft-ui-xaml/pull/4424#discussion_r588446070) [](ancestors = 588446070)